py/torch_tensorrt/_compile.py

```diff
@@ -525,11 +530,19 @@ def save(
             if not retrace:
                 from torch_tensorrt.dynamo._exporter import export
 
-                exp_program = export(module, arg_inputs, kwarg_inputs)
+                if arg_inputs is not None:
```

Comment:
same here

Reply:
changed to warning instead of raising error